packages/contracts/package.json

```diff
@@ -21,7 +21,7 @@
         "build": "ts-node ./scripts/buildVerifiers.ts && yarn compile && yarn abi && hardhat typechain && tsc",
         "abi": "node ./scripts/abi",
         "compile": "hardhat compile",
-        "test": "hardhat test ./test/*.test.ts",
+        "test": "hardhat test ./test/unirepVoting.test.ts",
```

Comment:
please revert the change of this line 🙏🏻

Reply:
whats the best way to just run one test? i tried `hardhat test ./test/unirep.test.ts` but got command not found

Reply:
you can either change the `test` script in your local to be like this (but not commit this change)
or you can run `hardhat test $FILE_PATH` in the root `packages/contracts` or `packages/core`
since in root directory there is no `hardhat` installed